queenbee/schema/function.py

```diff
@@ -22,92 +132,123 @@ class Function(BaseModel):
         ' this function.'
     )
 
-    inputs: Arguments = Field(
-        None,
+    operator: str = Field(
+        ...,
+        description='Function operator name.'
+    )
+    
+    inputs: FunctionInputs = Field(
+        FunctionInputs(),
         description=u'Input arguments for this function.'
     )
 
     command: str = Field(
-        ...,
+        '...',
```

Comment:
This shouldn't be a string. The original one looks correct to me.

Reply:
Nope, using `...` without quotes means that this is a mandatory field and has no default value. if we use `"..."` then we are saying that if you do not input a command into your function then the command = ...

Reply:
I see. But why would you want your default command to be `...`?

Reply:
I don't. The existing set up just means this field is mandatory.

Reply:
Man we were in agreement the whole time. I misread the changes here for ones that you were proposing rather than ones I had made :see_no_evil: